cromwell_tools/cromwell_tools.py

```diff
@@ -208,3 +219,58 @@ def read_local_file(path):
     with open(path) as f:
         contents = f.read()
     return contents
+
+
+def validate_cromwell_label(label_object):
```

Comment:
What happens if you submit a label to Cromwell that violates these rules? Could we save some effort here by just submitting what the user gives us and returning the Cromwell error message if the labels are bad? Would be preferable not to have to duplicate the validation that Cromwell does, especially since the rules might change in future.

Reply:
Yeah, I thought about leaving the work to Cromwell itself. The problem here is that, different from using the PATCH label endpoint(Cromwell will intercept invalid labels right away), adding labels with the submit endpoint will let Cromwell starts the workflow and processes the inputs until it finds the invalid labels. This will end up with some "broken-like" workflows in Cromwell, such as 8d5dc88b-6ebd-4478-8955-5ca7bb4064e3 in the staging env (JMUI will show a no-title workflow for this) So I turned to add a validator before the submission, although this cost more effort to develop and maintain, it brings a lot of early-warning advantages to us.

Reply:
>  it brings a lot of early-warning advantages to us

Big 👍 here. I'm of the opinion that if we can avoid sending things to Cromwell that it won't accept, it is worth a small maintenance cost. 

O/T: Ideally we would have a validator that would check the whole input package (including dependencies.json, inputs.json) and validate that your run is actually going to execute, with errors deriving strictly from the `command` blocks. This is more of a development ask than a Lira ask, though.